src/store/index.ts

```diff
@@ -40,6 +40,8 @@ export class CalChartState extends Serializable<CalChartState> {
 
   showDotLabels = true;
 
+  gridSize = 2;
```

Comment:
Once the x2 comment is resolved in `GrapherField`, let's update this
```suggestion
  gridSize = 4;
```